src/js/player.js

```diff
@@ -840,6 +843,8 @@ class Player extends Component {
       techOptions[props.getterName] = this[props.privateName];
     });
 
+    assign(techOptions, this.options_[titleTechName]);
```

Comment:
Going to add some unit tests to assign to find out, will open another PR with those